python_orocos_kdl/tests/dynamicstest.py

```diff
@@ -0,0 +1,74 @@
+# Copyright  (C)  2007  Ruben Smits <ruben dot smits at mech dot kuleuven dot be>
+
+# Version: 1.0
+# Author: Ruben Smits <ruben dot smits at mech dot kuleuven dot be>
```

Comment:
@MatthijsBurgh If this is a new file that you wrote, you should add the info here.